bdsx/bds/blockpos.ts

```diff
@@ -22,69 +22,69 @@ export namespace Facing {
 @nativeClass()
 export class BlockPos extends NativeStruct {
     @nativeField(int32_t)
-    x:int32_t;
+    x: int32_t;
```

Comment:
[Code Contribution Guideline 7
Avoid changing the format of old code whenever possible](https://github.com/bdsx/bdsx/blob/master/CONTRIBUTING.md#6-avoid-multiple-accessing-of-the-native-field)

Reply:
fixed